server/models.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
+const productDetailsModel = require ('../db/index.js').productDetails;
+
+class productDetails {
+  constructor() {
+    this.model = productDetailsModel;
+  }
+}
```

Comment:
Don't forget to add new lines at the end of every file! (or have the linter do it for you)